manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TELUS.TELUSBusinessConnectPhone
+PackageVersion: 23.1.1.45971
+PackageLocale: en-US
+Publisher: TELUS
+PackageName: TELUS Business Connect Phone
+License: http://telus.com/BusinessConnect/ServiceTerms

```

```suggestion
License: Proprietary
LicenseUrl: http://telus.com/BusinessConnect/ServiceTerms
```
A URL is not a License